### PR TITLE
Add Laravel Homestead to list of Dev Environments

### PR DESCRIPTION
**Pull request by damcclean**

### CODE REVIEW

**Comment by jackmcdade** [comment]
Is it best for windows?

**Comment by jasonvarga** [comment]
"Works well for Windows, too."

**Comment by damcclean** [comment]
It was the only thing I could find that Statamic worked on when I was working on Windows. 👍

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Wide-spread dev environments that work well (e.g. MAMP)
 
 - [MAMP 3](http://www.mamp.info/en/) (*MAMP 2 works as well*)
 - [Laravel Valet](https://laravel.com/docs/master/valet)
+- [Laravel Homestead](https://laravel.com/docs/master/homestead) - best for Windows
```
[the remark this answers is not in the document]
Comment (jasonvarga) [reply README.md]: and change the text to "Works well for Windows"

**Comment by damcclean** [comment]
Updated

**Pull request merged**